internal/pkg/service/stream/dependencies/reqbranch_test.go

```diff
@@ -0,0 +1,180 @@
+package dependencies
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/jarcoal/httpmock"
+	"github.com/keboola/go-client/pkg/keboola"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/definition/key"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
+)
+
+func TestBranchRequestScope_DefaultBranch_String(t *testing.T) {
+	t.Parallel()
+
```

Comment:
Test for different cases: branch ID can be `"default"`, `"456"` (ID of default branch), `"456"` (ID of not default branch).